apps/storybook-react/stories/Menu.stories.jsx

```diff
@@ -223,7 +223,12 @@ export const EdgeDetection = () => {
       >
         Top left
       </FloatingAnchor>
-      <Menu id="menu-topleft" open={Boolean(topLeft)} anchorEl={topLeft}>
+      <Menu
+        id="menu-topleft"
+        open={Boolean(topLeft)}
+        anchorEl={topLeft}
+        onClose={onClick}
```

Comment:
& dis

Reply:
What dis @mimarz? 😁

Reply:
If it’s the name of the event handler you’re referring to, I would call that something else than `onClick` since it’s being called by other events as well.

Reply:
…preferably something that starts with `handle` since that’s the convention we agreed on way back some time :wink: `handleClick`, `handleChange` etc.

Reply:
This syntax may come in handy 🤷 https://github.com/equinor/design-system/blob/develop/apps/storybook-react/stories/Accordion/Accordion.stories.jsx#L119-L122